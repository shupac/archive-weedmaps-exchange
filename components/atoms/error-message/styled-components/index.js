/* eslint-disable import/prefer-default-export */
/**
 * Auto-generated by Moonshot-CLI on Thu Sep 21 2017 09:45:10 GMT-0700 (PDT)
 */

export { default as Wrapper } from './wrapper.styled';
