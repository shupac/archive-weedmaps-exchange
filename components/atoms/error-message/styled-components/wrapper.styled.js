import styled from 'styled-components';
import { rem } from 'polished';

const Wrapper = styled.div`
  padding: 0 15px;

  h1 {
    display: flex;
    justify-content: center;
    font-size: ${rem(42)};
    margin: 0;
    padding-top: 64px;
    padding-bottom: 24px;
  }

  div {
    font-weight: 500;
    font-size: ${rem(20)};
    text-align: center;
  }
`;

export default Wrapper;
