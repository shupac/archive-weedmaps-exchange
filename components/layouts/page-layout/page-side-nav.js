import React from 'react';
import styled from 'styled-components';

// UI Components
import { SideNav, SideNavLayout, SideNavLink, AppHeader } from '@ghostgroup/ui';

import { getPath } from '@/lenses/navigation';

// Component Data
import {
  buyerData,
  sellerData,
  buyerFooterData,
  sellerFooterData,
} from './menuData';

// Styled Components
const { SideNavHeader, SideNavFooter } = SideNavLayout;
const Wrapper = styled.div`
  align-self: stretch;
`;

class SideNavComponent extends React.Component {
  render() {
    const { collapse } = this.props;

    const linksData = asBuyer ? buyerData : sellerData;
    const footerData = asBuyer ? buyerFooterData : sellerFooterData;

    return (
      <Wrapper>
        <SideNav collapse={collapse}>
          <SideNavHeader>
            <AppHeader
              title="exchange"
              onClick={e => {
                e.preventDefault();
              }}
            />
          </SideNavHeader>

          {this.renderNavLinks(linksData)}

          <SideNavFooter>{this.renderNavLinks(footerData)}</SideNavFooter>
        </SideNav>
      </Wrapper>
    );
  }

  renderNavLinks(data) {
    //const { dispatch, activePath } = this.props;

    return data.map(({ name, icon, path }) => (
      <SideNavLink
        key={name}
        name={name}
        icon={icon}
        //isActive={matchPath(activePath, path)}
        //onClick={() => dispatch(push(path))}
      />
    ));
  }
}

export default SideNavComponent
