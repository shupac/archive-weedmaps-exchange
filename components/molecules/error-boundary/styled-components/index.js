export { default as ErrorWrapper } from './wrapper.styled';
export { default as ErrorTitle } from './title.styled';
