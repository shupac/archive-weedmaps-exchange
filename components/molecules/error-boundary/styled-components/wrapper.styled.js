import styled from 'styled-components';
import { rem } from 'polished';

const Wrapper = styled.div`
  padding: ${rem(16)};
`;

export default Wrapper;
