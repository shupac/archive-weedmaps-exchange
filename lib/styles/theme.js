import { rem } from 'polished';
import { WmTheme } from '@ghostgroup/ui';

export default {
  style: WmTheme.style,
  // Design Enforced colors
  palette: {
    white: '#FFFFFF',
    teal: '#00CDBE',
    darkTeal: '#50BAB3',
    blue: '#4A90E2',
    purple: '#936DC7',
    red: '#D0021B',
    yellow: '#F5A623',
    green: '#00B359',
    darkGrey1: '#9FA9BA',
    darkGrey2: '#727682',
    lightGrey1: '#E6EAEE',
    lightGrey2: '#CBD1E0',
    lightGrey3: '#F6F7FA',
    lightGrey4: '#FAFBFC',
    lightGrey5: '#F1F4F7',
    darkBlue1: '#354052',
    darkBlue2: '#262935',
    darkBlue3: '#1D1F26',
  },
  colors: {
    primary: '#00CDBE',
    // Accent
    border: '#cccccc',
    lightTeal: '#68E6DD',
    teal: '#00CDBE',
    midTeal: '#01A8A3',
    darkTeal: '#005F5C',
    forestGreen: '#004D47',
    persianGreen: '#02A99D',
    elfGreen: '#15957C',
    pink: '#FF3366',
    darkPink: '#DB0045',
    blueHaze: '#CBD1E0',
    dodgerBlue: '#0066FF',
    bondiBlue: '#0093B6',
    denimBlue: '#1B8AC9',
    skyBlue: '#68D6DD',
    fountainBlue: '#50BAB3',
    havelockBlue: '#4A90E2',
    oxfordBlue: '#354052',
    amethyst: '#936DC7',
    sushiGreen: '#71AD30',
    porcelain: '#F2F5F5',
    red: '#D0021B',
    denyRed: '#D95350',
    approveGreen: '#5DB75C',
    pendingYellow: '#FFB200',
    buttercup: '#F5A623',
    divider: '#E6EAEE',
    headerGrey: '#727682',
    lightGrey3: '#F6F7FA',

    // Grayscale
    white: '#FFFFFF',
    snow: '#FEFEFE',
    ghost: '#FDFDFD',
    vapor: '#F4F5F5',
    whiteSmoke: '#F1f1f1',
    smoke: '#EEEEEE',
    mercury: '#E6E5E5',
    silver: '#DFDFDF',
    gainsboro: '#DDDDDD',
    iron: '#CCCCCC',
    aluminum: '#999999',
    gullGray: '#9FA9BA',
    athensGray: '#FAFBFC',
    steel: '#666666',
    charcoal: '#4A4A4A',
    oil: '#333333',
    jet: '#222222',
    tar: '#111111',
    black: '#000000',

    bg: {
      grayGradient: `linear-gradient(0deg,#F2F4F7 0%,#FFFFFF 100%)`,
      error: 'rgba(208,2,27,0.1)',
    },

    shadow: {
      light: 'rgba(0,0,0,0.2)',
      dark: 'rgba(0,0,0,0.7)',
    },

    // Listing Levels
    listingLevelGold: '#FFB700',
    listingLevelSilver: '#0E93B4',
    listingLevelBronze: '#E07000',

    // Navigation Menu
    navMenuBackgroundActive: '#1A1B23',
    navMenuBackgroundInactive: '#262935',
  },
  navigation: {
    sideNavWidth: rem(220),
    sideNavWidthCollapsed: rem(66),
  },
  dimensions: {
    // in pixels
    desktopHeader: 75,
    mobileHeader: 45,
    globalPadding: 16,
  },
  text: {
    fontFamily:
      '-apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",sans-serif',
    proximaNovaFont:
      'proxima-nova, -apple-system, BlinkMacSystemFont, "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif',
  },
  // Breakpoints, in em units
  breakpoints: [40, 52, 64],
  breakpointNames: ['xs', 'sm', 'md', 'lg'],
};
