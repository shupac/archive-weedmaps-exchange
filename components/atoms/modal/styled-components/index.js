import styled from 'styled-components';
import theme from 'lib/styles/theme';
import { rem } from 'polished';

export const ModalContainer = styled.div`
  background-color: ${theme.colors.white};
  border-radius: 3px;
  position: relative;
  pointer-events: auto;
  min-width: 200px;
  max-height: 1000px;
  overflow-y: scroll;
`;

export const CloseButton = styled.a`
  position: absolute;
  cursor: pointer;
  top: 16px;
  right: 16px;
`;

export const ModalBackdrop = styled.div`
  position: absolute;
  width: 100%;
  height: 100%;
`;

export const ModalHeader = styled.h4`
  border-bottom: 1px solid ${theme.style.icon.inverted};
  padding: ${rem(16)};
  margin: 0;
  user-select: none;
`;
export const ModalOverlay = styled.div`
  padding: 60px;
`;
