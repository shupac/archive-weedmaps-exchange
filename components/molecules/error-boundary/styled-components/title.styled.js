import styled from 'styled-components';
import { rem } from 'polished';
import { color } from 'lib/styles/theme-getters';

const Title = styled.span`
  height: ${rem(22)};
  color: ${color('denyRed')};
  font-size: ${rem(16)};
  font-weight: 500;
  line-height: ${rem(16)};
`;

export default Title;
