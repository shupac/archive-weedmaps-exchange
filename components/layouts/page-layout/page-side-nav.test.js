import { shallow } from 'enzyme';
import { ThemeProvider } from 'styled-components';
import toJSON from 'enzyme-to-json';
import AuthStore from 'lib/stores/auth';
import theme from 'lib/styles/theme';
import { PageSideNav } from './page-side-nav';

describe('The PageSideNav', () => {
  it('should be able to show ops_manager role only sidenav', () => {
    const auth = AuthStore.createStore({
      user: {
        roles: ['ops_manager'],
      },
    });
    const tree = toJSON(
      shallow(
        <ThemeProvider theme={theme}>
          <PageSideNav
            collapse={false}
            flags={{}}
            auth={auth}
            router={{ route: '/deals' }}
          />
        </ThemeProvider>,
      ).dive(),
    );

    expect(tree).toMatchSnapshot();
  });

  it('should be able to show all links sidenav when not taxes route', () => {
    const auth = AuthStore.createStore({
      user: {
        roles: [
          'ops_manager',
          'sales_manager',
          'content_moderator',
          'listing_owner',
        ],
      },
    });
    const tree = toJSON(
      shallow(
        <ThemeProvider theme={theme}>
          <PageSideNav
            collapse={false}
            flags={{}}
            auth={auth}
            router={{ route: '/deals' }}
          />
        </ThemeProvider>,
      ).dive(),
    );

    expect(tree).toMatchSnapshot();
  });

  it('should be able to show all links sidenav, except for feature flagged ones', () => {
    const auth = AuthStore.createStore({
      user: {
        roles: [
          'ops_manager',
          'sales_manager',
          'content_moderator',
          'listing_owner',
        ],
      },
    });
    const tree = toJSON(
      shallow(
        <ThemeProvider theme={theme}>
          <PageSideNav
            collapse={false}
            flags={{
              'admin-nearby-placements-link': false,
            }}
            auth={auth}
            router={{ route: '/deals' }}
          />
        </ThemeProvider>,
      ).dive(),
    );

    expect(tree).toMatchSnapshot();
  });
});
