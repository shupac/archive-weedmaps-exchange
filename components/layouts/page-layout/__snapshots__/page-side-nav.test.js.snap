// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The PageSideNav should be able to show all links sidenav when not taxes route 1`] = `
<SideNav
  collapse={false}
>
  <SideNavHeader>
    <AppHeader
      title="admin"
    />
  </SideNavHeader>
  <TaxLinkFence
    collapse={false}
    currentRoute="/deals"
  >
    <WithTheme(SideNavLink)
      collapse={false}
      href="https://weedmaps.com/new_admin/"
      icon={[Function]}
      name="Back To Listings"
      style={
        Object {
          "whiteSpace": "nowrap",
        }
      }
    />
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="taxes"
    />
  </TaxLinkFence>
  <NavLinkFence
    currentRoute="/deals"
  >
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="listings"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/dispensaries"
      >
        My Dispensaries
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/deliveries"
      >
        My Deliveries
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/doctors"
      >
        My Doctors
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/converted_listings"
      >
        Convert Listings
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="brands"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands"
      >
        Brands
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/banners"
      >
        Banners
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/categories"
      >
        Categories
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/brand_placements"
      >
        Brand Placements
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/product_placements"
      >
        Product Placements
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/reports/brands"
      >
        Reports Placements
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="deals"
    >
      <ChildLink
        active={false}
        href="/deals/placement"
      >
        Featured Deals
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/deals/moderate"
      >
        Moderate Deals
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/region_deal_subscriptions"
      >
        Daily Deal Email
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="reporting"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/moderation/reported_contents"
      >
        Reported Contents
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/blacklist_incidents"
      >
        Blacklist Incidents
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="learn"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/strains"
      >
        Strains
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/medical_uses"
      >
        Medical Uses
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/cannabinoids"
      >
        Pharmacopeia
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="ops"
    >
      <ChildLink
        active={false}
        href="https://feature-flag-admin-acceptance.internal-weedmaps.com"
      >
        Feature Flag Service
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/rollout"
      >
        Feature Flag (Rollout)
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/regions"
      >
        Manage Regions
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/delivery_service_areas"
      >
        Delivery Service Areas
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/banned_ips"
      >
        IP Ban Tool
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/users"
      >
        Users
      </ChildLink>
    </WithTheme(SideNavLink)>
  </NavLinkFence>
</SideNav>
`;

exports[`The PageSideNav should be able to show all links sidenav, except for feature flagged ones 1`] = `
<SideNav
  collapse={false}
>
  <SideNavHeader>
    <AppHeader
      title="admin"
    />
  </SideNavHeader>
  <TaxLinkFence
    collapse={false}
    currentRoute="/deals"
  >
    <WithTheme(SideNavLink)
      collapse={false}
      href="https://weedmaps.com/new_admin/"
      icon={[Function]}
      name="Back To Listings"
      style={
        Object {
          "whiteSpace": "nowrap",
        }
      }
    />
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="taxes"
    />
  </TaxLinkFence>
  <NavLinkFence
    currentRoute="/deals"
  >
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="listings"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/dispensaries"
      >
        My Dispensaries
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/deliveries"
      >
        My Deliveries
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/doctors"
      >
        My Doctors
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/converted_listings"
      >
        Convert Listings
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="brands"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands"
      >
        Brands
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/banners"
      >
        Banners
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/categories"
      >
        Categories
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/brand_placements"
      >
        Brand Placements
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/product_placements"
      >
        Product Placements
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/reports/brands"
      >
        Reports Placements
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="deals"
    >
      <ChildLink
        active={false}
        href="/deals/placement"
      >
        Featured Deals
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/deals/moderate"
      >
        Moderate Deals
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/region_deal_subscriptions"
      >
        Daily Deal Email
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="reporting"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/moderation/reported_contents"
      >
        Reported Contents
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/blacklist_incidents"
      >
        Blacklist Incidents
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="learn"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/strains"
      >
        Strains
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/medical_uses"
      >
        Medical Uses
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/cannabinoids"
      >
        Pharmacopeia
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="ops"
    >
      <ChildLink
        active={false}
        href="https://feature-flag-admin-acceptance.internal-weedmaps.com"
      >
        Feature Flag Service
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/rollout"
      >
        Feature Flag (Rollout)
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/regions"
      >
        Manage Regions
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/delivery_service_areas"
      >
        Delivery Service Areas
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/banned_ips"
      >
        IP Ban Tool
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/users"
      >
        Users
      </ChildLink>
    </WithTheme(SideNavLink)>
  </NavLinkFence>
</SideNav>
`;

exports[`The PageSideNav should be able to show ops_manager role only sidenav 1`] = `
<SideNav
  collapse={false}
>
  <SideNavHeader>
    <AppHeader
      title="admin"
    />
  </SideNavHeader>
  <TaxLinkFence
    collapse={false}
    currentRoute="/deals"
  >
    <WithTheme(SideNavLink)
      collapse={false}
      href="https://weedmaps.com/new_admin/"
      icon={[Function]}
      name="Back To Listings"
      style={
        Object {
          "whiteSpace": "nowrap",
        }
      }
    />
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="taxes"
    />
  </TaxLinkFence>
  <NavLinkFence
    currentRoute="/deals"
  >
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="listings"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/dispensaries"
      >
        My Dispensaries
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/deliveries"
      >
        My Deliveries
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/doctors"
      >
        My Doctors
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/converted_listings"
      >
        Convert Listings
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="brands"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands"
      >
        Brands
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/banners"
      >
        Banners
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/categories"
      >
        Categories
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/brand_placements"
      >
        Brand Placements
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/product_placements"
      >
        Product Placements
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/brands/reports/brands"
      >
        Reports Placements
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="deals"
    >
      <ChildLink
        active={false}
        href="/deals/placement"
      >
        Featured Deals
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/deals/moderate"
      >
        Moderate Deals
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/region_deal_subscriptions"
      >
        Daily Deal Email
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="reporting"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/moderation/reported_contents"
      >
        Reported Contents
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/blacklist_incidents"
      >
        Blacklist Incidents
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="learn"
    >
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/strains"
      >
        Strains
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/medical_uses"
      >
        Medical Uses
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/cannabinoids"
      >
        Pharmacopeia
      </ChildLink>
    </WithTheme(SideNavLink)>
    <WithTheme(SideNavLink)
      collapse={false}
      icon={[Function]}
      isActive={false}
      name="ops"
    >
      <ChildLink
        active={false}
        href="https://feature-flag-admin-acceptance.internal-weedmaps.com"
      >
        Feature Flag Service
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/rollout"
      >
        Feature Flag (Rollout)
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/regions"
      >
        Manage Regions
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/delivery_service_areas"
      >
        Delivery Service Areas
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/banned_ips"
      >
        IP Ban Tool
      </ChildLink>
      <ChildLink
        active={false}
        href="https://weedmaps.com/new_admin/users"
      >
        Users
      </ChildLink>
    </WithTheme(SideNavLink)>
  </NavLinkFence>
</SideNav>
`;
